docker/images/pinot/etc/jmx_prometheus_javaagent/configs/server.yml

```diff
@@ -73,6 +66,14 @@ rules:
     table: "$1$3"
     tableType: "$4"
     partition: "$5"
+- pattern: "\"org\\.apache\\.pinot\\.common\\.metrics\"<type=\"ServerMetrics\", name=\"pinot\\.server\\.endToEndRealtimeIngestionDelayMs\\.(([^.]+)\\.)?([^.]*)_(OFFLINE|REALTIME)\\.(\\w+)\"><>(\\w+)"
```

Comment:
`realtimeIngestionDelayMs` is a different metric than this. Can you validate if that one is being successfully exported?

Reply:
Yeah. `realtimeIngestionDelayMs` is handled by moving the metric rule earlier present on line 16 to line 175.
Will add screenshots on validation as well